### ISSUE TITLE
RecipeSage in a subfolder

### ISSUE THREAD

**Issue by cristobalvp** [julianpoy/RecipeSage-selfhost#19]
Hi,
 Thanks for sharing this project, it looks great.

 I am trying to configure my reverse proxy to be able to access RecipeSage from Internet, but I am having some issues because I use subfolders instead of subdomains.

 I tried to change proxy.conf file, but it is not working. Is there some way to change the URL Base in RecipeSage, so I can access through https://mydomain.com/RecipeSage?

Thanks again!

**Comment by julianpoy** [comment]
proxy.conf shouldn't need to be modified AFAIK.

The main issue I see is that the frontend is always going to look for the backend at /api by default. You can specify `API_BASE_OVERRIDE = '/custom-api-location-here/'` on the window to override the location of the API, but you'll need to find a way to do that for every pageload. You could always extend the frontend container with a dockerfile and add that to the index.html :)

**Comment by cristobalvp** [comment]
Thanks. I will continue trying to make it works. My initial test did not work, but I am missing something since all the requests are still going to /api.

**Comment by da-wi** [comment]
> Thanks. I will continue trying to make it works. My initial test did not work, but I am missing something since all the requests are still going to /api.

Did you find a solution for this configuration?

**Comment by julianpoy** [comment]
@da-wi I added a script in the latest commit that facilitates this. You can just set `API_BASE_OVERRIDE` for the static container to whatever the publicly accessible URL for the API is.

**Comment by da-wi** [comment]
Thank you! I tried to run the new commit behind my reverse proxy at example.com/recipesage. Most of the static content can now be accessed, after additionally changing <base href="/"> to the new root <base "/recipesage/">. However, even after running the entrypoint script, provisioning of some static assets fails (specifically, assets loaded from polyfills with static links, i.e., background images). I did not have time to look into it further, but maybe somebody else has an idea to make this work.
